### PR TITLE
Remove active sync on the master side

### PR DESCRIPTION
**Pull request by jiacheliu3**
### What changes are proposed in this pull request?

Remove active sync feature including RPCs and relevant commands.

### Why are the changes needed?

The file metadata and all related features are no longer needed at the Master. This is one step is removing FileSystemMaster/BlockMaster.

### Does this PR introduce any user facing changes?

The property keys and corresponding commands are removed.

### CODE REVIEW

**Comment by jiacheliu3** [comment]
fyi @yuzhu

**Review by yyongycy — APPROVED** [review]
lgtm, please also propagate the proto change

**Review by JiamingMai — APPROVED** [review]
LGTM

File: dora/underfs/hdfs/pom.xml [hunk 1]
```diff
@@ -135,6 +129,17 @@
   </build>
 
   <profiles>
+    <profile>
```
Comment (jiacheliu3) [inline dora/underfs/hdfs/pom.xml:132]: i had to sort this after we require poms to be sorted. No idea why there wasn't this problem before I touched it...

**Review by yyongycy — APPROVED** [review]
lgtm

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**